windnode_abw/model/region/model.py

```diff
@@ -335,10 +340,10 @@ def create_el_model(region=None, datetime_index=None):
                     v_level='hv' if row['v_nom'] == 110 else 'ehv'
                 ),
                 outputs={bus: solph.Flow(
-                    variable_costs=region.tech_assumptions_scn.loc[
-                        'elenergy']['capex'],
-                    emissions=region.tech_assumptions_scn.loc[
-                        'elenergy']['emissions_var']
+                    variable_costs=(costs_var + emissions_var *
+                                    region.tech_assumptions_scn.loc[
+                                        'emission']['capex']),
+                    emissions=emissions_var
```

Comment:
Does the assignment of emission to a Flow have any effect on the optimization? Since we optimize for minimal costs, emissions should play a role... Or is it simply to bring this to parameters that can be extracted from oemof-solph results?